app/views/repositories/new.html.erb

```diff
@@ -38,6 +38,10 @@
 
     <div id="image-container"></div>
 
+      <%= f.label :youtube_link, "Youtube Link" %>
+      <div>Your project will be better understood with a 1 min video Youtube.<br>Votre projet sera mieux compris avec une vidéo de 1 min Youtube</div>
```

Comment:
1 min Youtube video:)

Reply:
Thank you! Parastoo! I just corrected it ;)